python/flask/secure-static-file-serve.yaml

```diff
@@ -0,0 +1,14 @@
+# For Flask app to serve file securely, it should use `send_from_directory` method
+# The other methods like `send_file` or `send_static_file` are not recommended per
+# https://stackoverflow.com/questions/20646822/how-to-serve-static-files-in-flask
+rules:
+  - id: avoid_send_file_without_path_sanitization
+    patterns:
+      - pattern-inside: |
+          def $X(filename):
```

Comment:
Ideally we would have decorator support here https://github.com/returntocorp/sgrep/issues/29